KeeperSdk/vault/RecordTypeCommands.cs

```diff
@@ -205,13 +243,92 @@ public void checkAdminAccess()
                     throw new VaultException("Enterprise admin access is required for this command");
                 }
             }
+
+            public List<InputRecordType> ValidateRecordTypeFile(string filePath)
```

Comment:
This method does not belong to SDK space.
What is the usage of this method inside SDK?